content/03.methods.md

```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
 CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
 We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] v.2.0.23 on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh).
 
+#### Focal Copy Number Calling
+
+We added the ploidy inferred via ControlFreeC to the CNVkit and consensus CNV segmentation files and used the ploidy and copy number values to define a status column with gain and loss values broadly.
```

Comment:
```suggestion
We added the ploidy inferred via Control-FREEC to the CNVkit and consensus CNV segmentation files and used the ploidy and copy number values to define a status column with gain and loss values broadly.
```
Noticed this one when I was adding the spell checker

Reply:
You don't need to say status _column_. You do need to be specific about what "gain and loss values broadly" means.